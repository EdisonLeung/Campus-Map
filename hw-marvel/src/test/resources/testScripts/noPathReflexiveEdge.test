# Tests pathfinding where there is no path and the nodes goes in a loop. Purpose of
# the test is to test that the pathfinding will not create an infinite loop

# Create graph
CreateGraph noPathReflexive

# Add nodes
AddNode noPathReflexive a
AddNode noPathReflexive b
AddNode noPathReflexive c

# Add edges
AddEdge noPathReflexive a b AB
AddEdge noPathReflexive b a BA

# Pathfinding from a to c
FindPath noPathReflexive a c