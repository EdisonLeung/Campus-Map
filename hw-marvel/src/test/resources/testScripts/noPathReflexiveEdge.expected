# Tests pathfinding where there is no path and the nodes goes in a loop. Purpose of
# the test is to test that the pathfinding will not create an infinite loop

# Create graph
created graph noPathReflexive

# Add nodes
added node a to noPathReflexive
added node b to noPathReflexive
added node c to noPathReflexive

# Add edges
added edge AB from a to b in noPathReflexive
added edge BA from b to a in noPathReflexive

# Pathfinding from a to c
path from a to c:
no path found